packages/containers/src/ComponentForm/ComponentForm.selectors.js

```diff
@@ -0,0 +1,16 @@
+import get from 'lodash/get';
+import Immutable from 'immutable';
+import { TCompForm } from './ComponentForm.component';
+
+export function getComponentFormState(state, componentName) {
+	return get(state, ['cmf', 'components'], Immutable.Map()).getIn(
+		[TCompForm.displayName, componentName],
+		Immutable.Map(),
+	);
+}
+
+export function isComponentFormDirty(state, componentName) {
+	return getComponentFormState(state, componentName).get('dirty', false);
```

Comment:
so here you can do
```
return TCompForm.getState(state, componentName).get('dirty', false)
```

And by default is use the default state

Reply:
Thx for this feedback, i make the changes ;)